Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0612 Acc: 0.1160
val Loss: 2.8101 Acc: 0.2121

Epoch 1/19
----------
train Loss: 2.5231 Acc: 0.3770
val Loss: 2.3723 Acc: 0.3868

Epoch 2/19
----------
train Loss: 2.0677 Acc: 0.5074
val Loss: 2.0953 Acc: 0.4308

Epoch 3/19
----------
train Loss: 1.7068 Acc: 0.6143
val Loss: 1.9092 Acc: 0.4868

Epoch 4/19
----------
train Loss: 1.3995 Acc: 0.6852
val Loss: 1.7658 Acc: 0.5165

Epoch 5/19
----------
train Loss: 1.1820 Acc: 0.7377
val Loss: 1.6811 Acc: 0.5275

Epoch 6/19
----------
train Loss: 0.9569 Acc: 0.7973
val Loss: 1.6074 Acc: 0.5604

Epoch 7/19
----------
train Loss: 0.7855 Acc: 0.8485
val Loss: 1.5399 Acc: 0.5582

Epoch 8/19
----------
train Loss: 0.6545 Acc: 0.8813
val Loss: 1.4986 Acc: 0.5560

Epoch 9/19
----------
train Loss: 0.5270 Acc: 0.9186
val Loss: 1.4580 Acc: 0.5692

Epoch 10/19
----------
train Loss: 0.4239 Acc: 0.9426
val Loss: 1.4161 Acc: 0.5868

Epoch 11/19
----------
train Loss: 0.3491 Acc: 0.9567
val Loss: 1.4042 Acc: 0.5846

Epoch 12/19
----------
train Loss: 0.2829 Acc: 0.9658
val Loss: 1.4327 Acc: 0.5703

Epoch 13/19
----------
train Loss: 0.2360 Acc: 0.9790
val Loss: 1.3640 Acc: 0.5868

Epoch 14/19
----------
train Loss: 0.1954 Acc: 0.9842
val Loss: 1.3868 Acc: 0.5879

Epoch 15/19
----------
train Loss: 0.1743 Acc: 0.9891
val Loss: 1.4219 Acc: 0.5868

Epoch 16/19
----------
train Loss: 0.1391 Acc: 0.9939
val Loss: 1.3778 Acc: 0.5835

Epoch 17/19
----------
train Loss: 0.1313 Acc: 0.9939
val Loss: 1.3357 Acc: 0.6011

Epoch 18/19
----------
train Loss: 0.1161 Acc: 0.9956
val Loss: 1.3678 Acc: 0.5956

Epoch 19/19
----------
train Loss: 0.0944 Acc: 0.9982
val Loss: 1.3503 Acc: 0.5967

Training complete in 21m 56s
Best val Acc: 0.601099
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.05_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 60.110 Acc@5 88.901

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.05_kar_LongRange.png'

Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0680 Acc: 0.1270
val Loss: 2.8140 Acc: 0.2615

Epoch 1/19
----------
train Loss: 2.4736 Acc: 0.4015
val Loss: 2.3794 Acc: 0.3670

Epoch 2/19
----------
train Loss: 2.0163 Acc: 0.5280
val Loss: 2.0741 Acc: 0.4538

Epoch 3/19
----------
train Loss: 1.6456 Acc: 0.6340
val Loss: 1.8875 Acc: 0.4901

Epoch 4/19
----------
train Loss: 1.3599 Acc: 0.7075
val Loss: 1.7584 Acc: 0.5165

Epoch 5/19
----------
train Loss: 1.1497 Acc: 0.7539
val Loss: 1.6700 Acc: 0.5341

Epoch 6/19
----------
train Loss: 0.9369 Acc: 0.8078
val Loss: 1.5824 Acc: 0.5495

Epoch 7/19
----------
train Loss: 0.7830 Acc: 0.8371
val Loss: 1.5255 Acc: 0.5527

Epoch 8/19
----------
train Loss: 0.6509 Acc: 0.8765
val Loss: 1.4690 Acc: 0.5681

Epoch 9/19
----------
train Loss: 0.5379 Acc: 0.9067
val Loss: 1.4546 Acc: 0.5692

Epoch 10/19
----------
train Loss: 0.4240 Acc: 0.9400
val Loss: 1.4192 Acc: 0.5802

Epoch 11/19
----------
train Loss: 0.3768 Acc: 0.9488
val Loss: 1.4069 Acc: 0.5791

Epoch 12/19
----------
train Loss: 0.3100 Acc: 0.9606
val Loss: 1.3467 Acc: 0.6044

Epoch 13/19
----------
train Loss: 0.2601 Acc: 0.9711
val Loss: 1.3669 Acc: 0.5967

Epoch 14/19
----------
train Loss: 0.2113 Acc: 0.9847
val Loss: 1.3539 Acc: 0.6110

Epoch 15/19
----------
train Loss: 0.1714 Acc: 0.9908
val Loss: 1.3477 Acc: 0.6099

Epoch 16/19
----------
train Loss: 0.1515 Acc: 0.9921
val Loss: 1.3069 Acc: 0.6066

Epoch 17/19
----------
train Loss: 0.1303 Acc: 0.9908
val Loss: 1.3173 Acc: 0.6011

Epoch 18/19
----------
train Loss: 0.1142 Acc: 0.9939
val Loss: 1.3082 Acc: 0.6319

Epoch 19/19
----------
train Loss: 0.0972 Acc: 0.9956
val Loss: 1.3590 Acc: 0.6165

Training complete in 21m 31s
Best val Acc: 0.631868
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.1_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 63.187 Acc@5 88.132

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.1_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0352 Acc: 0.1454
val Loss: 2.7811 Acc: 0.2934

Epoch 1/19
----------
train Loss: 2.4429 Acc: 0.4483
val Loss: 2.3077 Acc: 0.4187

Epoch 2/19
----------
train Loss: 1.9131 Acc: 0.6025
val Loss: 1.9537 Acc: 0.5044

Epoch 3/19
----------
train Loss: 1.4990 Acc: 0.6948
val Loss: 1.6985 Acc: 0.5648

Epoch 4/19
----------
train Loss: 1.1814 Acc: 0.7614
val Loss: 1.5191 Acc: 0.5835

Epoch 5/19
----------
train Loss: 0.9379 Acc: 0.8244
val Loss: 1.3731 Acc: 0.6066

Epoch 6/19
----------
train Loss: 0.7442 Acc: 0.8656
val Loss: 1.2785 Acc: 0.6418

Epoch 7/19
----------
train Loss: 0.5998 Acc: 0.8997
val Loss: 1.1853 Acc: 0.6462

Epoch 8/19
----------
train Loss: 0.4810 Acc: 0.9286
val Loss: 1.1418 Acc: 0.6670

Epoch 9/19
----------
train Loss: 0.3842 Acc: 0.9501
val Loss: 1.1002 Acc: 0.6747

Epoch 10/19
----------
train Loss: 0.3173 Acc: 0.9623
val Loss: 1.0482 Acc: 0.6780

Epoch 11/19
----------
train Loss: 0.2614 Acc: 0.9715
val Loss: 1.0290 Acc: 0.6868

Epoch 12/19
----------
train Loss: 0.2099 Acc: 0.9838
val Loss: 0.9951 Acc: 0.6912

Epoch 13/19
----------
train Loss: 0.1785 Acc: 0.9860
val Loss: 0.9927 Acc: 0.7000

Epoch 14/19
----------
train Loss: 0.1668 Acc: 0.9860
val Loss: 0.9796 Acc: 0.7011

Epoch 15/19
----------
train Loss: 0.1306 Acc: 0.9947
val Loss: 0.9661 Acc: 0.7066

Epoch 16/19
----------
train Loss: 0.1113 Acc: 0.9939
val Loss: 0.9494 Acc: 0.7088

Epoch 17/19
----------
train Loss: 0.1005 Acc: 0.9956
val Loss: 0.9254 Acc: 0.7154

Epoch 18/19
----------
train Loss: 0.0933 Acc: 0.9961
val Loss: 0.9617 Acc: 0.7055

Epoch 19/19
----------
train Loss: 0.0894 Acc: 0.9947
val Loss: 0.9263 Acc: 0.7187

Training complete in 21m 6s
Best val Acc: 0.718681
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.3_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 71.868 Acc@5 94.396

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.3_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0209 Acc: 0.1725
val Loss: 2.7127 Acc: 0.2956

Epoch 1/19
----------
train Loss: 2.3682 Acc: 0.4877
val Loss: 2.1642 Acc: 0.4670

Epoch 2/19
----------
train Loss: 1.7788 Acc: 0.6484
val Loss: 1.7511 Acc: 0.5615

Epoch 3/19
----------
train Loss: 1.3470 Acc: 0.7566
val Loss: 1.4631 Acc: 0.6242

Epoch 4/19
----------
train Loss: 1.0065 Acc: 0.8249
val Loss: 1.2648 Acc: 0.6703

Epoch 5/19
----------
train Loss: 0.7823 Acc: 0.8660
val Loss: 1.0830 Acc: 0.7176

Epoch 6/19
----------
train Loss: 0.6047 Acc: 0.9063
val Loss: 0.9618 Acc: 0.7407

Epoch 7/19
----------
train Loss: 0.4704 Acc: 0.9352
val Loss: 0.9034 Acc: 0.7407

Epoch 8/19
----------
train Loss: 0.3715 Acc: 0.9545
val Loss: 0.8291 Acc: 0.7593

Epoch 9/19
----------
train Loss: 0.3057 Acc: 0.9628
val Loss: 0.7739 Acc: 0.7670

Epoch 10/19
----------
train Loss: 0.2518 Acc: 0.9729
val Loss: 0.7447 Acc: 0.7736

Epoch 11/19
----------
train Loss: 0.2030 Acc: 0.9847
val Loss: 0.7074 Acc: 0.7802

Epoch 12/19
----------
train Loss: 0.1616 Acc: 0.9895
val Loss: 0.6767 Acc: 0.7945

Epoch 13/19
----------
train Loss: 0.1419 Acc: 0.9877
val Loss: 0.6682 Acc: 0.7923

Epoch 14/19
----------
train Loss: 0.1165 Acc: 0.9926
val Loss: 0.6403 Acc: 0.8011

Epoch 15/19
----------
train Loss: 0.1051 Acc: 0.9956
val Loss: 0.6485 Acc: 0.7956

Epoch 16/19
----------
train Loss: 0.0913 Acc: 0.9978
val Loss: 0.6241 Acc: 0.8055

Epoch 17/19
----------
train Loss: 0.0749 Acc: 0.9969
val Loss: 0.6247 Acc: 0.8066

Epoch 18/19
----------
train Loss: 0.0790 Acc: 0.9965
val Loss: 0.6079 Acc: 0.8154

Epoch 19/19
----------
train Loss: 0.0684 Acc: 0.9982
val Loss: 0.6076 Acc: 0.8044

Training complete in 21m 13s
Best val Acc: 0.815385
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.5_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 81.538 Acc@5 97.143

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.5_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 3.0149 Acc: 0.1642
val Loss: 2.6836 Acc: 0.3440

Epoch 1/19
----------
train Loss: 2.2519 Acc: 0.5613
val Loss: 1.9336 Acc: 0.5956

Epoch 2/19
----------
train Loss: 1.5537 Acc: 0.7723
val Loss: 1.3985 Acc: 0.7176

Epoch 3/19
----------
train Loss: 1.0507 Acc: 0.8489
val Loss: 1.0324 Acc: 0.7758

Epoch 4/19
----------
train Loss: 0.7206 Acc: 0.9094
val Loss: 0.8055 Acc: 0.8022

Epoch 5/19
----------
train Loss: 0.5170 Acc: 0.9304
val Loss: 0.6568 Acc: 0.8352

Epoch 6/19
----------
train Loss: 0.3739 Acc: 0.9580
val Loss: 0.5730 Acc: 0.8538

Epoch 7/19
----------
train Loss: 0.2925 Acc: 0.9702
val Loss: 0.5065 Acc: 0.8582

Epoch 8/19
----------
train Loss: 0.2291 Acc: 0.9777
val Loss: 0.4656 Acc: 0.8703

Epoch 9/19
----------
train Loss: 0.1817 Acc: 0.9877
val Loss: 0.4063 Acc: 0.8780

Epoch 10/19
----------
train Loss: 0.1599 Acc: 0.9873
val Loss: 0.3970 Acc: 0.8857

Epoch 11/19
----------
train Loss: 0.1254 Acc: 0.9930
val Loss: 0.3749 Acc: 0.8901

Epoch 12/19
----------
train Loss: 0.1189 Acc: 0.9921
val Loss: 0.3437 Acc: 0.9033

Epoch 13/19
----------
train Loss: 0.1098 Acc: 0.9891
val Loss: 0.3320 Acc: 0.9044

Epoch 14/19
----------
train Loss: 0.0894 Acc: 0.9952
val Loss: 0.3311 Acc: 0.9000

Epoch 15/19
----------
train Loss: 0.0769 Acc: 0.9952
val Loss: 0.3144 Acc: 0.9132

Epoch 16/19
----------
train Loss: 0.0694 Acc: 0.9974
val Loss: 0.2906 Acc: 0.9176

Epoch 17/19
----------
train Loss: 0.0670 Acc: 0.9956
val Loss: 0.3188 Acc: 0.9000

Epoch 18/19
----------
train Loss: 0.0569 Acc: 0.9974
val Loss: 0.2934 Acc: 0.9154

Epoch 19/19
----------
train Loss: 0.0567 Acc: 0.9969
val Loss: 0.2946 Acc: 0.9066

Training complete in 21m 17s
Best val Acc: 0.917582
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.7_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 91.758 Acc@5 99.560

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.7_kar_LongRange.png'
Use 4 GPUs!
{'train': 'Num of clips:2284', 'val': 'Num of clips:910'}
Epoch 0/19
----------
train Loss: 2.8875 Acc: 0.2583
val Loss: 2.3659 Acc: 0.4967

Epoch 1/19
----------
train Loss: 1.9044 Acc: 0.7119
val Loss: 1.4337 Acc: 0.7912

Epoch 2/19
----------
train Loss: 1.1411 Acc: 0.8914
val Loss: 0.9095 Acc: 0.8692

Epoch 3/19
----------
train Loss: 0.6729 Acc: 0.9387
val Loss: 0.6082 Acc: 0.9088

Epoch 4/19
----------
train Loss: 0.4602 Acc: 0.9615
val Loss: 0.4635 Acc: 0.9176

Epoch 5/19
----------
train Loss: 0.3131 Acc: 0.9750
val Loss: 0.3622 Acc: 0.9286

Epoch 6/19
----------
train Loss: 0.2336 Acc: 0.9847
val Loss: 0.3136 Acc: 0.9374

Epoch 7/19
----------
train Loss: 0.1885 Acc: 0.9886
val Loss: 0.2899 Acc: 0.9385

Epoch 8/19
----------
train Loss: 0.1547 Acc: 0.9891
val Loss: 0.2657 Acc: 0.9341

Epoch 9/19
----------
train Loss: 0.1244 Acc: 0.9961
val Loss: 0.2403 Acc: 0.9418

Epoch 10/19
----------
train Loss: 0.1088 Acc: 0.9917
val Loss: 0.2207 Acc: 0.9462

Epoch 11/19
----------
train Loss: 0.0946 Acc: 0.9952
val Loss: 0.2111 Acc: 0.9462

Epoch 12/19
----------
train Loss: 0.0847 Acc: 0.9943
val Loss: 0.1996 Acc: 0.9484

Epoch 13/19
----------
train Loss: 0.0782 Acc: 0.9961
val Loss: 0.2006 Acc: 0.9505

Epoch 14/19
----------
train Loss: 0.0714 Acc: 0.9947
val Loss: 0.1807 Acc: 0.9549

Epoch 15/19
----------
train Loss: 0.0584 Acc: 0.9978
val Loss: 0.1868 Acc: 0.9516

Epoch 16/19
----------
train Loss: 0.0613 Acc: 0.9961
val Loss: 0.1739 Acc: 0.9538

Epoch 17/19
----------
train Loss: 0.0517 Acc: 0.9996
val Loss: 0.1795 Acc: 0.9549

Epoch 18/19
----------
train Loss: 0.0507 Acc: 0.9991
val Loss: 0.1646 Acc: 0.9516

Epoch 19/19
----------
train Loss: 0.0436 Acc: 0.9978
val Loss: 0.1692 Acc: 0.9560

Training complete in 20m 56s
Best val Acc: 0.956044
Saved the weights of the best epoch in /home/acb11711tx/lzq/ExpEval/model_param/ucf101_24_r2p1d_16_Full_g_0.9_kar_LongRange.pt
Num of clips:910
 Val: Acc@1 95.604 Acc@5 100.000

Normalized confusion matrix
Traceback (most recent call last):
  File "model_train/retrain_ucf101_24_model.py", line 172, in <module>
    cm_fig.savefig(os.path.join(proj_root, 'visual_res', 'cm', f'{save_label}.png'))
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    with cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 626, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/home/acb11711tx/miniconda3/envs/pytorch1.1/lib/python3.6/site-packages/matplotlib/cbook/__init__.py", line 611, in to_filehandle
    fh = io.open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: '/home/acb11711tx/lzq/ExpEval/visual_res/cm/ucf101_24_r2p1d_16_Full_g_0.9_kar_LongRange.png'
